### DISCUSSION TITLE
Additional training

### DISCUSSION THREAD

**Discussion by peregilk** [openai/whisper#32] [Q&A] [answered]
Have you released any code making it possible to do additional pretraining/finetuning? This is of course especially relevant for the languages that have high WER today but where it is possible to obtain additional training data.

**Comment by jongwook** [comment]
We currently don't have plans to release training/fine-tuning code, but there might be an implementation from the community soon.

**Comment by sanchit-gandhi** [comment, marked as answer]
And here it is! Check-out this blog for fine-tuning Whisper for multilingual ASR with Hugging Face Transformers: https://huggingface.co/blog/fine-tune-whisper

It provides a step-by-step guide to fine-tuning, right from data preparation to evaluation 🤗 There'a Google Colab so you can also run it as a notebook 😉
